packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -130,22 +130,35 @@ export default class RelationControl extends React.Component {
     const valueField = field.get('valueField');
     const displayField = field.get('displayFields') || field.get('valueField');
 
-    return hits.map(hit => {
-      let labelReturn;
-      if (List.isList(displayField)) {
-        labelReturn = displayField
-          .toJS()
-          .map(key => this.parseNestedFields(hit, key))
-          .join(' ');
-      } else {
-        labelReturn = this.parseNestedFields(hit, displayField);
-      }
-      return {
-        data: hit.data,
-        value: this.parseNestedFields(hit, valueField),
-        label: labelReturn,
-      };
-    });
+    let data = hits;
```

Comment:
Variable naming here is aweful, but I have a hard time coming up with something better. Perhaps this whole block could be rewritten to be better